backend/main.go

```
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"github.com/kjanat/poo-tracker/backend/internal/repository"
+	"github.com/kjanat/poo-tracker/backend/internal/service"
+	"github.com/kjanat/poo-tracker/backend/server"
+)
+
+func main() {
+	app := server.New(repository.NewMemory(), service.AvgBristol{})
+	app.Engine.Run() // defaults to :8080
```

golangci-lint (1.64.8)

Error return value of `app.Engine.Run` is not checked

(errcheck)